package.json

```diff
@@ -173,6 +173,7 @@
     "babel-loader": "8.2.5",
     "babel-plugin-angularjs-annotate": "0.10.0",
     "babel-plugin-macros": "3.1.0",
+    "blob-polyfill": "7.0.20220408",
```

Comment:
I added that in order to add methods to the `Blob` object, which seem to be missing on JSDom.

Ref https://github.com/jsdom/jsdom/issues/2555